pkgs/applications/networking/instant-messengers/bitlbee-mastodon/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchgit, stdenv, bitlbee, autoreconfHook, pkgconfig, glib }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "bitlbee-mastodon-${version}";
```

Comment:
Please replace with `pname = "bitlbee-mastodon";` as `pname = "${name}-${version}";` is already implied.

Reply:
I didn't know about that, done.